File: app/src/main/java/com/chelomin/wallaby/Wallaby.java
package com.chelomin.wallaby;

import android.app.Application;

/**
 * Created by huge on 12/11/17.
 */

public class Wallaby extends Application {
    private static Wallaby instance;

    @Override
    public void onCreate() {
        super.onCreate();

        instance = this;
    }

    public static Wallaby getInstance() {
        return instance;
    }
}

File: app/src/main/java/com/chelomin/wallaby/Wallaby.kt
package com.chelomin.wallaby

import android.app.Application

/**
 * Created by huge on 12/11/17.
 */

class Wallaby : Application() {

    override fun onCreate() {
        super.onCreate()

        instance = this
    }

    companion object {
        lateinit var instance: Wallaby
            private set
    }
}
